utilities/test_suite/HIP/runTests.py

```diff
@@ -27,6 +28,24 @@ def validate_path(input_path):
     if not os.path.isdir(input_path):
         raise ValueError("path " + input_path + " is not a directory.")
 
+def create_layout_directories(dst_path, layout_dict):
+    for layout in range(3):
+        current_layout = layout_dict[layout]
+        try:
+            os.makedirs(dst_path + '/' + current_layout)
+        except FileExistsError:
+            pass
+        folder_list = [f for f in os.listdir(dst_path) if current_layout.lower() in f]
+        for folder in folder_list:
+            os.rename(dst_path + '/' + folder, dst_path + '/' + current_layout +  '/' + folder)
+
+def get_log_file_list():
+    return [
+        "../OUTPUT_PERFORMANCE_LOGS_HOST_NEW/Tensor_host_pkd3_raw_performance_log.txt",
```

Comment:
This is the HIP test suite file. Why is it returning HOST times?

Reply:
Please ensure HIP/HOST times we are getting here are validated against any times we had in the previous test suite.